src/zodbupdate/tests.py

```diff
@@ -1266,11 +1293,76 @@ def test_loaded_renames_override_missing_interfaces(self):
         self.assertEqual({}, renames)
 
 
-def test_suite():
-    suite = unittest.TestSuite()
-    suite.addTest(unittest.makeSuite(TestsBasics))
-    if six.PY2:
-        suite.addTest(unittest.makeSuite(Python2Tests))
-    if six.PY3:
-        suite.addTest(unittest.makeSuite(Python3Tests))
-    return suite
+class RelStorageHFMixin(object):
+    """
+    Mixin to create a history-free RelStorage using SQLite.
+    """
+    def _makeStorage(self):
+        from ZODB import config
+        return config.storageFromString(
+            """
+            %%import relstorage
+            <relstorage>
+                keep-history false
+                blob-dir %s
+                <sqlite3>
+                   data-dir %s
+                </sqlite3>
+            </relstorage>
+            """ % (os.path.join(self.tmpblob, 'blobs'), self.tmpblob,)
```

Comment:
Minor: I'd've used "%import ... blob-dir {blobdir} ... ".format(blobdir=...) to avoid having to double the percents.